api/model/Organization.ts

```diff
@@ -1,13 +1,37 @@
 import { SchemaDefinition as def } from '@contember/schema-definition'
 import { Model } from '@contember/schema'
 import { Offer } from './Offer'
+import { District } from './District'
 
+export const OrganizationTypeEnum = def.createEnum(
+	'collegeInitiative',
+	'researchAndUniversitySector',
+	'governmentOrganization',
+	'privateOrganization',
+	'other',
+	'osvcPerson',
+	'municipality',
+	'nonprofit',
+	'foundation',
+	'media',
+	'church',
+	'volunteerInitiative'
+)
+
+// TODO: okres, kraj
 export class Organization {
 	name = def.stringColumn().notNull()
+	address = def.stringColumn().notNull()
+	identificationNumber = def.stringColumn().notNull()
+	website = def.stringColumn().notNull()
+	note = def.stringColumn().nullable()
+	parentOrganization = def.stringColumn().nullable()
```

Comment:
Za mě je otázka, zda to nemá být vazba.

Reply:
To za me taky. Je to na diskusi s produktem, resp. se Zdenkem, nicmene je to dobrej point